package.json

```diff
@@ -34,10 +34,10 @@
     "@aeternity/bip39": "^0.1.0",
     "@aeternity/hd-wallet": "^0.2.0",
     "@stamp/it": "^1.1.0",
-    "@zxing/library": "^0.17.1",
+    "@zxing/library": "^0.18.3",
```

Comment:
There were some breaking changes, I don't think that it can be updated easily

Reply:
I have checked it, it works on my Android build.

Reply:
this library is used in a web version, `cordova-plugin-qrscanner` is used in Cordova